server/server.go

```diff
@@ -177,6 +177,8 @@ func (s *Server) handle(conn *gokeyless.Conn) {
 		case gokeyless.OpECDSASignSHA1:
 			opts = crypto.SHA1
 		case gokeyless.OpRSASignSHA224:
+			isRSA = true
```

Comment:
can we group the condition in which isRSA is assigned true? the sequence of alternative 'fallthrough's makes it a little hard to maintain in my opinion.

have second switch on Opcode {
case gokeyless.OpRSASignSHA224, gokeyless.OpRSASignSHA256, gokeyless.OpRSASignSHA384, gokeyless.OpRSASignSHA512:
isRSA=true
}

and did opts assignment like
case gokeyless.OpRSASignSHA224, gokeyless.OpECDSASignSHA224:
opts = crypto.SHA224
case gokeyless.OpRSASignSHA256, gokeyless.OpECDSASignSHA256:
...